include/common_structs.h

```diff
@@ -279,9 +279,9 @@ typedef struct {
     /* 0x0010 */ s16 currentItemCopy; // Has no effect on what item the players has, It is just a synced copy
     /* 0x0012 */ s16 unk_012;
     /* 0x0014 */ Vec3f pos;
-    /* 0x0020 */ f32 copy_rotation_x;
-    /* 0x0024 */ f32 copy_rotation_y;
-    /* 0x0028 */ f32 copy_rotation_z;
+    /* 0x0020 */ f32 copy_x; // or old position?
+    /* 0x0024 */ f32 copy_y;
+    /* 0x0028 */ f32 copy_z; 
```

Comment:
Just wondering if you confirmed that these are not rotation vectors?

Reply:
of what I see sometime it can be rotation sometime it's the old position but it's just use as copy

Reply:
the problem it's not only pos in some case it's rotation so it use like a temporary copy

Reply:
> in some case it's rotation
I looked and it's never used as rotation as far as I can tell. I think that was just due to old names.

Reply:
hmm I look also and yeah never use rotation maybe I mix with another variable